easybuild/easyconfigs/s/SearchGUI/SearchGUI-3.3.3-Java-1.8.0_152.eb

```diff
@@ -0,0 +1,39 @@
+easyblock = 'PackedBinary'
+
+name = 'SearchGUI'
+version = '3.3.3'
+versionsuffix = '-Java-%(javaver)s'
+
+homepage = 'https://github.com/compomics/searchgui'
+description = """SearchGUI is a user-friendly open-source graphical user interface for configuring and 
+                 running proteomics identification search engines and de novo sequencing algorithms, 
+                 currently supporting X! Tandem, MS-GF+, MS Amanda, MyriMatch, Comet, Tide, Andromeda, OMSSA,
+                 Novor and DirecTag."""
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+source_urls = ['http://genesis.ugent.be/maven2/eu/isas/searchgui/SearchGUI/%(version)s/']
+sources = ['SearchGUI-%(version)s-mac_and_linux.tar.gz']
+
+
+dependencies = [
+    ('Java', '1.8.0_152'),
+    ('Mono', '4.6.2.7')
+]
+
+
+sanity_check_paths = {
+    'files': ['SearchGUI-%(version)s.jar'],
+    'dirs': ['lib', 'resources'],
+}
+
+
+modaliases = {
+    'SearchCLI': 'java -cp $EBROOTSEARCHGUI/SearchGUI-%(version)s.jar eu.isas.searchgui.cmd.SearchCLI',
```

Comment:
@korbini Is this using some kind of mirror, or is that just the way how `SearchCLI` works?

Reply:
It looks for me this is like it works, so we found it convenient to give it an version independent alias.
https://compomics.github.io/projects/searchgui/wiki/searchcli.html